README.rst

```diff
@@ -280,6 +280,15 @@ It is highly recommended to use the NVIDIA PyTorch or NeMo container if having i
 
 Transformer Engine requires PyTorch to be built with CUDA 11.8.
 
+
+Flash Attention
+~~~~~~~~~~~~~~~~~~~~
+Install flash-attn <https://github.com/HazyResearch/flash-attention> if you want to use Flash Attention to reduce memory overhead. 
+.. code-block:: bash
+
+  pip install flash-attn
+  pip install triton==2.0.0.dev20221202
+
```

Comment:
Can you also clarify that if the TransformerEngine path is used, it will use it's own version of Flash Attention?

Reply:
Currently, TransformerEngine also use the same open source Flash Attention. They don't have their own version.  The difference is whether we implement FA in NeMo or TransformerEngine.

Reply:
Right but if user has `TransformerEngine=True` and `use_flash_attention=False` they will still be using flash attention, for example. Better to help clarify this

Reply:
Added